src/org/joni/Region.java

```diff
@@ -33,6 +33,7 @@ public static Region newRegion(int begin, int end) {
         return new SingleRegion(begin, end);
     }
 
+    @Override
     public abstract Region clone();
```

Comment:
does this one really emit a compilation warning? it's weird to tag given it being abstract ...

Reply:
Yes, it overrides `Object#clone()`